__tests__/app.test.js

```diff
@@ -165,10 +164,48 @@ describe("GET reviews ", () => {
             owner: expect.any(String),
             review_body: expect.any(String),
```

Comment:
Can you double check the Trello Board instructions for this line 165 ... it doesn't require you to bring the body back as part of the api and database call. You'll need remove review_body through your code.